spec/features/hub/create_organization_hierarchy_spec.rb

```diff
@@ -28,26 +28,63 @@
 
       expect(page).to have_text("No sites")
 
-      fill_in "Name", with: "Oregano Org"
-      select "Coati Coalition", from: "Coalition"
-      click_on "Save"
+      within "#organization-form" do
+        fill_in "Name", with: "Oregano Org"
+        select "Coati Coalition", from: "Coalition"
+        click_on "Save"
+      end
       expect(page).to have_selector("h1", text: "Oregano Org")
 
+      # adding / removing zip codes
+      within "#zip-code-routing-form" do
+        fill_in "Zip code", with: "94606"
+        click_on "Save"
+        Capybara.using_wait_time(10) do
```

Comment:
Did you find that you needed the increased wait time? Just curious - if so, we could consider raising it everywhere. :/

Reply:
yes i did, intermittantly. We could raise it everywhere. It only has an impact if it actually needs to wait longer.